megamek/src/megamek/common/Entity.java

```diff
@@ -3018,6 +3018,14 @@ public boolean isBoardProhibited(int mapType) {
         return false;
     }
 
+    /**
+     * Returns true if the specified hex contains some sort of deadly
+     * terrain.
+     */
+    public boolean isLocationDeadly(Coords c) {
```

Comment:
I assume implementing this for other unit types is a "future development" type thing?

Reply:
yea, also to make it similar to isLocationProhibited()